### PR TITLE
Adding other saucelabs browsers as allowed failures.

### PR DESCRIPTION
**Pull request by mithro**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,17 +9,24 @@ env:
...
-    - BROWSER=Remote ARGS='-b Remote --sauce --remote-caps=INTERNETEXPLORER --remote-caps=version=10 --remote-caps=platform="Windows 8"'
-#    - BROWSER=Remote ARGS='-b Remote --sauce --remote-caps=SAFARI --remote-caps=version=6 --remote-caps=platform="OS X 10.8"'
-#    - BROWSER=Remote ARGS='-b Remote --sauce --remote-caps=IPHONE --remote-caps=version=6 --remote-caps=platform="OS X 10.8"'
-#    - BROWSER=Remote ARGS='-b Remote --sauce --remote-caps=ANDROID --remote-caps=version=4.0 --remote-caps=platform=Linux'
+#    - BROWSER=Chrome-stable ARGS='-x -b Chrome -u'
+#    - BROWSER=Chrome-beta ARGS='-x -b Chrome -u'
+#    - BROWSER=Chrome-unstable ARGS='-x -b Chrome -u'
+#    - BROWSER=Firefox-stable ARGS='-x -b Firefox -u'
+#    - BROWSER=Firefox-beta ARGS='-x -b Firefox -u'
+#    - BROWSER=Firefox-aurora ARGS='-x -b Firefox -u'
```
Comment (alancutter) [inline .travis.yml]: Should these tests be commented out?

Comment (mithro) [inline .travis.yml]: Fixed!

**Comment by mithro** [comment]
PTAL.

**Comment by alancutter** [comment]
lgtm

**Pull request merged**